### DISCUSSION TITLE
error: pattern matching in instanceof is not supported in -source 11       if (props.get(Snapshot.TRANSFORM_MATRIX) instanceof ReadableNativeArray matrixArray) {^   (use -source 16 or higher to enable pattern matching in instanceof)

### DISCUSSION THREAD

**Discussion by TanPhoi** [software-mansion/react-native-reanimated#6620] [Q&A]
C:\Users\devta\ReactNativeProjects\hello_app\node_modules\react-native-reanimated\android\src\main\java\com\swmansion\reanimated\layoutReanimation\AnimationsManager.java:356: error: pattern matching in instanceof is not supported in -source 11
      if (props.get(Snapshot.TRANSFORM_MATRIX) instanceof ReadableNativeArray matrixArray) {
                                                                              ^
  (use -source 16 or higher to enable pattern matching in instanceof)
C:\Users\devta\ReactNativeProjects\hello_app\node_modules\react-native-reanimated\android\src\main\java\com\swmansion\reanimated\NodesManager.java:492: error: switch rules are not supported in -source 11
        case Boolean -> copy.pushBoolean(array.getBoolean(i));
                     ^
  (use -source 14 or higher to enable switch rules)
2 errors

**Comment by Asadaial** [comment]
going through this issue and stuck from last 4 days

**Reply by beyzanuryanar** [reply]
Did you solve it? @Asadaial

**Reply by TanPhoi** [reply]
> Did you solve it? @Asadaial

You need update react native reanimated version up "3.7.2"

**Reply by TanPhoi** [reply]
> going through this issue and stuck from last 4 days

You need update react native reanimated version up "3.7.2"

**Comment by jaferi512** [comment]
Facing the same issue on React Native 0.71.19 version.
Following
